web/html/src/components/tree/tree.stories.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Tree } from './tree';
+
+const simpleTreeData = {
+  rootId: "2",
+  items: [
+    {id: "2", children: ['3', '4', '8']},
+    {id: "3", data: {key: "three", description: "I am 3"}},
+    {id: "4", data: {key: "for", description: "I am 4 children"}, children: ["5", "6", "7"]},
```

Comment:
`s/for/four/`

The `I am 4 children` and the same for the following sounds a little weird. I'ld rather see something like `I am #4, with children`